src/com/facebook/buck/cli/MissingSymbolsHandler.java

```diff
@@ -221,6 +221,6 @@ private String formatTarget(BuildTarget buildTarget) {
   }
 
   private void print(String line) {
-    console.getStdOut().println(console.getAnsi().asWarningText(line));
+    console.getStdErr().println(console.getAnsi().asWarningText(line));
```

Comment:
Unfortunately this doesn't work :/

SuperConsole does a dirty check before every render, and the fact that the build failed has already dirtied the console.  We need SuperConsole to keep printing `ConsoleEvents` even after stderr is dirty.